playground/mocks/data/idp/idx/success.json

```diff
@@ -1,32 +1,43 @@
 {
-    "stateHandle": "022FJBvS6bHS7OjQSOhtfghhJvO2a2TELaMZ94s69K",
+    "stateHandle": "02EQcms7FOAByCjwRqpUCKfzxNJj2wJ_nh3x7zheyD",
```

Comment:
Instead of updating this view.. just create a new one for your usecase. for eg `success-with-email-identifier`

Reply:
What about responseConfig.js? Now SuccessView won't work with the original version of success.json because the API response in there is incomplete.

Reply:
? slack.